data/sql/updates/db_world/0000_00_00_00.sql

```diff
@@ -0,0 +1,459 @@
+
+
+-- [Q:25212] Vent Horizon 
+UPDATE `conditions` SET `ConditionValue1` = '25212' WHERE (`SourceTypeOrReferenceId`='15') AND (`SourceGroup`='11211') AND (`SourceEntry`='0') AND (`ElseGroup`='0') AND (`ConditionTypeOrReference`='9') AND (`ConditionValue1`='25283') AND (`ConditionValue2`='0') AND (`ConditionValue3`='0');
+DELETE FROM `creature_template_addon` WHERE (`entry` = '39420');
+INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `auras`) VALUES ('39420', '0', '0', '65536', '1', '0', '75779');
+UPDATE `creature_template` SET `scale` = '2' WHERE `entry` = '39420';
+DELETE FROM `conditions` WHERE `SourceEntry`=73082;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `ConditionTypeOrReference`, `ConditionValue1`, `ConditionValue2`) VALUES 
+('13', '1', '73082', '31', '3', '39420');
+
+-- [Q:25283] Prepping the Speech  
+UPDATE `creature_template` SET `AIName`='' WHERE `entry` IN(1268,7955,6119,39817);
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN(1268,7955,6119,39817); 
+UPDATE `creature_template` SET `ScriptName` = 'npc_mekkatorque', `scale` = '0.6', `unit_flags` = '33554752' WHERE `entry` = '39712';
+
+-- [Q:25286] Words for Delivery, we have 3, but "25286" is the correct one. 
+
+-- [Q:25393] Operation Gnomeregan 
+UPDATE `creature_template` SET `mechanic_immune_mask` = '12658704', `ScriptName` = 'npc_og_mekkatorque' WHERE `entry` = '39271';
+UPDATE `creature_template` SET `unit_flags` = '393220', `ScriptName` = 'npc_og_rl' WHERE `entry` = '39820';
+UPDATE `creature_template` SET `npcflag` = '0', `VehicleId` = '0', `ScriptName` = 'npc_og_tank' WHERE `entry` = '39860';
+UPDATE `creature_template` SET `faction` = '1771', `unit_flags` = '4', `ScriptName` = 'npc_og_cannon' WHERE `entry` = '39759';
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_bomber' WHERE `entry` = '39735';
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_infantry' WHERE `entry` = '39252';
+UPDATE `creature_template` SET `spell1` = '74764', `ScriptName` = 'npc_og_i_infantry' WHERE `entry` = '39755';
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_suit' WHERE `entry` = '39902';
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_trogg' WHERE `entry` IN ('39826', '39799');
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_boltcog' WHERE `entry` = '39837';
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_assistants' WHERE `entry` IN ('39273', '39910');
+UPDATE `creature_template` SET `ScriptName` = 'npc_og_i_tank' WHERE `entry` = '39819';
+UPDATE `creature_template` SET `AIName` = 'AggresorAI' WHERE `entry` IN ('39755', '39836');
+UPDATE `creature_template` SET `VehicleId` = '0', `ScriptName` = 'npc_og_camera_vehicle' WHERE `entry` = '40479';
+UPDATE `creature_template` SET `mechanic_immune_mask` = '8192' WHERE `entry` IN ('39860', '39826', '39799', '39819', '39273', '39910', '39837');
```

Comment:
1. I'd prefer if you add and remove flags using a=a|x a=a&~x
2. Not strings

Reply:
1)they are taken from TC as well as sniffs, so they are 100% correct and should not be modified in the future
2) fixed